cmd/demo/main.go

```diff
@@ -38,6 +37,13 @@ func main() {
 		},
 	)
 
+	server.AroundResponse = func(h http.Handler) http.Handler {
+		return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+			// Strip etag header from response
+			rw.Header().Del("etag")
```

Comment:
```suggestion
			rw.Header().Del("etag")
			h.ServeHTTP(rw, r)
```

Would this need to call `ServeHTTP` to serve the response?